### ISSUE TITLE
up-follow does not replace body in v0.51.0

### ISSUE THREAD

**Issue by dirkholzapfel** [unpoly/unpoly#49]
After upgrading to from v0.50.2 to v0.51.0 links with `up-follow` don't replace the body anymore.

The console log looks right, but the body is not updated.

````
[UP] Booting framework (up:framework:boot)
unpoly.min.js:2:1126
 __ _____  ___  ___  / /_ __
/ // / _ \/ _ \/ _ \/ / // /  0.51.0
\___/_//_/ .__/\___/_/\_. / 
        / /            / /

Call `up.log.disable()` to disable logging for this session.
unpoly.min.js:2:1126
[UP] Framework booted (up:framework:booted)
unpoly.min.js:2:1126
[UP] Booting user application (up:app:boot)
unpoly.min.js:2:1126
[UP] Compiling fragment  
<body style="height: 100%; margin-top: 8em;" up-source="http://localhost:8080/blog_page.html">
unpoly.min.js:2:1126
[UP] Inserted fragment  
<body style="height: 100%; margin-top: 8em;" up-source="http://localhost:8080/blog_page.html">
  (up:fragment:inserted ( 
Object { origin: undefined }
 ))
unpoly.min.js:2:1126
[UP] User application booted (up:app:booted)
unpoly.min.js:2:1126
[UP] Loading GET http://localhost:8080/uebermedien.html (up:proxy:load ( 
Object { request: {…} }
 ))
unpoly.min.js:2:1126
[UP] Server responded with HTTP 200 (58351 bytes) (up:proxy:loaded ( 
Object { request: {…}, response: {…} }
 ))
unpoly.min.js:2:1126
[UP] Extracting body from 58351 bytes of HTML
unpoly.min.js:2:1126
[UP] Adding history entry for http://localhost:8080/uebermedien.html (up:history:push ( 
Object { url: "http://localhost:8080/uebermedien.html" }
 ))
unpoly.min.js:2:1126
[UP] Advanced to location http://localhost:8080/uebermedien.html (up:history:pushed ( 
Object { url: "http://localhost:8080/uebermedien.html" }
 ))
unpoly.min.js:2:1126
[UP] Updating body
unpoly.min.js:2:1126
```

**Comment by triskweline** [comment]
Hey @dirkholzapfel . A few minutes ago I released 0.51.1 which should address this issue. Can you check?

**Comment by dirkholzapfel** [comment]
hi @triskweline . The problem is fixed with v0.51.1 on our side. thanks a lot for your fast response!

**Comment by triskweline** [comment]
:)
